data/70-browser.conf

```diff
@@ -0,0 +1 @@
+BROWSER=/usr/bin/sailfish-browser
```

Comment:
Should have invoker etc? Colon separated list of commands so I'd suppose it's ok to use: https://manpages.debian.org/stretch/man-db/man.1.en.html

Reply:
Was also pondering if this should be with or without /usr/bin/ see reference from https://cgit.freedesktop.org/xdg/xdg-utils/tree/scripts/xdg-open.in#n414

Not sure if it makes any difference and otherwise looks good to me, just pondering :)